src/content/user-guide/deepsparse-engine/logging.mdx

```diff
@@ -0,0 +1,319 @@
+(simple system performance analysis) to challenging (assessing the accuracy of the system in the wild by manually labeling 
+the input data distribution post-facto). Examples include:
+- System performance: what is the latency/throughput of a query?
+- Data quality: is there an issue getting data to my model?
+- Data distribution shift: does the input data distribution deviates over time to the point where the model stops to deliver reliable predictions?
+- Model accuracy: what is the percentage of correct predictions that a model achieves?
+
+DeepSparse Logging is designed to provide maximum flexibility for you to extract whatever data is needed from a 
+production inference pipeline into the logging system of your choice. 
+
+## Metrics 
+DeepSparse Logging provides access to two types of metrics.
+
+### System Logging Metrics 
+
+System Logging Metrics give you access to granular performance metrics for quick and efficient diagnosis of deployment system health.
+
+There is one group of System Logging Metrics currently available: Prediction Latency.  For each inference request, DeepSparse logs the following:
+1. Pre-processing Time - milliseconds in the pre-processing step
+2. Engine Time - milliseconds in the engine forward pass step
+3. Post-processing Time - milliseconds in the post-processing step
+4. Total Time - milliseconds for the end-to-end response time
+
+### Data Logging Metrics
+
+Data Logging Metrics give ML teams access to data at each stage of an inference pipeline. 
+This facilitates inspection of the data, understanding of its properties, detecting edge cases and possible data drift.
+
+There are 4 stages in the inference pipeline where Data Logging can occur:
+1. `pipeline_inputs`: raw input passed to the inference pipeline by the user
+2. `engine_inputs`: pre-processed tensors passed to the engine for the forward passed
+3. `engine_outputs`: result of the engine forward pass (e.g. the raw logits)
+4. `pipeline_outputs`: raw output returned to the pipeline caller
+
+At each stage, you can specify functions to be applied to the data before Logging. Examples functions include the identity function
+(for logging the raw input/output), the mean function (e.g. for monitoring the mean pixel value of an image), or a count function (e.g.
+for monitoring the number of unknown tokens in an NLP pipeline).
+
+There are three types of functions that can be applied to target data at each stage:
+1. Built-in functions - pre-written functions provided by DeepSparse ([see list on GitHub](https://github.com/neuralmagic/deepsparse/blob/main/src/deepsparse/loggers/metric_functions/built_ins.py)) 
+2. Framework Functions - functions from `torch` or `numpy` (e.g. `torch.mean`, `np.mean`)
+3. Custom Functions - custom user-provided functions with dynamic imports
+
+## Configuration
+
+A YAML file is used to configure both System and Data Logging.
+- **System Logging** is *enabled* if a `logger` is specified
```

Comment:
Right now the default behavior is, that even if the `logger` is not specified, the default logger gets created ( a single PythonLogger)